src/main/java/com/algorithmtron/model/StringDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.algorithmtron.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class StringDTO {
```

Comment:
Consider naming this DTO in a more appropriate matter to better reflect where it is used.

Reply:
Okay, i'll try to find a more suggestive name for it!